src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -84,26 +89,68 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * Prompts the users to grant or deny access to their current location.
    * @private 
   */
-  grantLocationAccess_() {
-    this.loadMap_();
-  }
+ grantLocationAccess_() {
+  this.loadMap_();
+}
 
-  /**
-   * Loads a map object that currently displays a hardcoded location.
-   * @param {!google.maps.LatLng=} coordinates The LatLng object of user's current location.
-   * TODO(kristenmason@): implement grant location access and modify
-   * function to pass in current position (currently coords default to GooglePlex)
-   * @private 
-  */
-  loadMap_(coordinates = new google.maps.LatLng(37.422004,-122.081402)) { 
-    const map = new google.maps.Map(document.getElementById('map'), {
-      zoom: DEFAULT_ZOOM,
-      center: coordinates
-    });
-    const marker = new google.maps.Marker({
-      position: coordinates,
-      map: map,
-      title: 'Current Position'
-    });
-  }
+/**
+ * Loads a map object that currently displays a hardcoded location.
+ * @param {!google.maps.LatLng=} coordinates The LatLng object of user's current location.
+ * @private 
+*/
+loadMap_(coordinates = new google.maps.LatLng(37.422004, -122.081402)) {
+  map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
is there a reason why this is not a const and declared here?

Reply:
Should've been fixed in a recent PR to declare as a member variable-- the map object gets used in other functions down the line so it has to be declared on a larger scope